tests/test_edgeql_data_migration.py

```diff
@@ -101,8 +114,38 @@ async def fast_forward_describe_migration(self, *, limit=None):
                     await self.con.execute('COMMIT MIGRATION;')
                     break
 
+                interpolations = {}
+
+                user_input_reqs = mig['proposed']['required_user_input']
+                if user_input_reqs:
+                    for var in user_input_reqs:
```

Comment:
Are these guaranteed to be in the right order? (I guess they might be, if `json` always respects the orderings?)

Reply:
Yes, `required_user_input` is an array, and we explicitly guarantee stable command order in diffs.